src/Forms/DiffField.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace SilverStripe\VersionedAdmin\Forms;
+
+use LogicException;
+use SilverStripe\Forms\FormField;
+use SilverStripe\Forms\HTMLReadonlyField;
+use SilverStripe\View\Parsers\Diff;
+
+class DiffField extends HTMLReadonlyField
```

Comment:
Can you please document clearly what this form field is for and provide an inline example of how to use it?